corim/testcases/regen-from-src.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/bash
+# Copyright 2024 Contributors to the Veraison project.
+# SPDX-License-Identifier: Apache-2.0
+set -e
+
+GEN_TESTCASE=$(go env GOPATH)/bin/gen-testcase
+
+if [[ ! -f ${GEN_TESTCASE} ]]; then
+	echo "installing gen-testcase"
+	go install github.com/veraison/gen-testcase@v0.0.1
+fi
```

Comment:
Just a curiosity: why didd we need a new tool to synthesise test cases rather than just using diag2cbor.rb as we do, e.g., in https://github.com/veraison/psatoken/tree/main/testvectors/cbor ?